.github/workflows/ci-containers.yaml

```
@@ -49,7 +44,7 @@ jobs:
         id: meta
         uses: docker/metadata-action@v4
         with:
-          images: ghcr.io/${{ github.repository }}/${{ matrix.container }}
+          images: ghcr.io/${{ github.repository }}/vivo
```

Are you sure you want ghcr.io/calyptia/vivo/vivo?
```suggestion:
          images: ghcr.io/${{ github.repository }}
```